### DISCUSSION TITLE
how could a start a mlua env as a REPL

### DISCUSSION THREAD

**Discussion by walksanatora** [mlua-rs/mlua#242] [Q&A] [answered]
i have some functions i want to test, but i perfer to use a repl
do i have to implement my own (by basically `load`ing strings constantly)

**Comment by khvzak** [comment, marked as answer]
You can use `repl` example from mlua:
```
❯ cargo run --example repl --features lua54,vendored
    Finished dev [unoptimized + debuginfo] target(s) in 0.21s
     Running `target/debug/examples/repl`
> print("hello, world")
hello, world

```

**Reply by walksanatora** [reply]
so i just have to copy the code from `repl` example into my project OK
